Agents/PropertyValueEstimationAgent/agent/entry_point.py

```diff
@@ -0,0 +1,53 @@
+# To avoid unnecessary logging information from py4j package, set logger level before 
+# first creation of JPS_BASE_LIB module view (i.e. jpsBaseLibView = jpsBaseLibGW.createModuleView())
+import logging
+logging.getLogger("py4j").setLevel(logging.ERROR)
```

Comment:
I think you can remove this line from here, the logger level is set here once-for-all (as long as you are importing `jpsBaseLibGW` from `pyderivationagent`): https://github.com/cambridge-cares/TheWorldAvatar/blob/900ac0aa13310de4031754ba536e57ff8e106484/JPS_BASE_LIB/python_derivation_agent/pyderivationagent/kg_operations/gateway.py#L9

Reply:
Addressed in [0c751ee](https://github.com/cambridge-cares/TheWorldAvatar/pull/454/commits/0c751eebbd9edecfea87c0e1f70074d634ab03c4)